### PR TITLE
Add BlockState::Accepted

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: sdk/src/types/api/core.rs [hunk 1]
```diff
@@ -339,31 +339,33 @@ pub struct SubmitBlockResponse {
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub enum BlockState {
-    // Stored but not confirmed.
+    /// Stored but not confirmed.
     Pending,
-    // Confirmed with the first level of knowledge.
+    /// TODO
```
Comment (DaughterOfMars) [inline sdk/src/types/api/core.rs]: TODO

File: bindings/python/iota_sdk/types/block/metadata.py [hunk 2]
```diff
@@ -33,16 +33,18 @@ class BlockState(Enum):
 
     Attributes:
         Pending: Stored but not confirmed.
+        Accepted: TODO
```
Comment (DaughterOfMars) [inline bindings/python/iota_sdk/types/block/metadata.py]: TODO

File: sdk/src/types/api/core.rs (hunk 1 above)
Comment (thibault-martinez) [reply sdk/src/types/api/core.rs]: This variant is not yet in the TIP 🤷🏻

Comment (DaughterOfMars) [reply sdk/src/types/api/core.rs]: probably should find a good description for it before merging tho

Comment (thibault-martinez) [reply sdk/src/types/api/core.rs]: Done

File: bindings/python/iota_sdk/types/block/metadata.py (hunk 2 above)
Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/block/metadata.py]: Done

**Review by DaughterOfMars — APPROVED** [review]

File: bindings/python/iota_sdk/types/block/metadata.py [hunk 3]
```diff
@@ -32,10 +32,10 @@ class BlockState(Enum):
     """Describes the state of a block.
 
     Attributes:
-        Pending: Stored but not confirmed.
-        Accepted: TODO
-        Confirmed: Confirmed with the first level of knowledge.
-        Finalized: Included and can no longer be reverted.
+        Pending: Stored but not accepted/confirmed.
+        Accepted: Valid block referenced by some validators.
```
Comment (Thoralf-M) [inline bindings/python/iota_sdk/types/block/metadata.py:36]: No better definition than "some"?

**Review by Thoralf-M — APPROVED** [review]

File: bindings/python/iota_sdk/types/block/metadata.py (hunk 3 above)
Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/block/metadata.py:36]: I think it's fine if we compare it to Confirmed we understand that it's less than 2/3 but I can add it if you want

Comment (Thoralf-M) [reply bindings/python/iota_sdk/types/block/metadata.py:36]: I just would myself understand what exactly this means, is a single reference from a single validator already enough to have it accepted?

Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/block/metadata.py:36]: I would say so, yes. I can draft and wait for a better comment 🤷🏻

Comment (Thoralf-M) [reply bindings/python/iota_sdk/types/block/metadata.py:36]: No need to block for this

**Pull request merged**